README.md

```diff
@@ -40,6 +40,9 @@ data class Listing<T>(
 Basically, you could manage all data streams with a `Listing` component, which is awesome!
 It's really flexible and useful to display the paged list entities and reflect the network status changes in the UI.
 
+Despite of this library was designed to work with paged endpoints, it **also supports** to work with **not paged endpoints**.
```

Comment:
maybe something like this?: `This library was designed to work with paged endpoints. However, it also supports working with not paged endpoints`.